docs/workforce-management-framework/topics/con-raincatcher-core-getting-started.adoc

```diff
@@ -0,0 +1,143 @@
+[[id='raincatcher-core-getting-started-{chapter}']]
+= RainCatcher-core Getting Started
+
+Getting started with the Feedhenry RainCatcher Core Repository
+
+== Developing
+
+RainCatcher supports and is tested on Node.JS@4 and should run on latest LTS.
```

Comment:
What do you think of introducing a variable for node version? This way we update it in one location and this update value will then appear everything the variable is used.